docs/WEATHER.md

```
@@ -0,0 +1,200 @@
+- `pressure` - Atmospheric pressure (hPa)
+- `windspeed` - Wind speed (m/s)
+- `winddirection` - Wind direction (degrees)
+- `precipitation` - Rainfall/snowfall (mm)
+- `solar_radiation` - Solar irradiance (W/m²)
+- `visibility` - Visibility distance (m)
+- `cloud_cover` - Cloud coverage (% or oktas)
+- `dew_point` - Dew point temperature (°C)
+
+## Command Options
+
+### Sequential Download
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)